libraries/rush-lib/src/cli/actions/CheckAction.ts

```diff
@@ -37,12 +38,29 @@ export class CheckAction extends BaseRushAction {
         'If this flag is specified, long lists of package names will not be truncated. ' +
         `This has no effect if the ${this._jsonFlag.longName} flag is also specified.`
     });
+    this._subspaceParameter = this.defineStringParameter({
+      parameterLongName: '--subspace',
+      argumentName: 'SUBSPACE_NAME',
+      description:
+        '(EXPERIMENTAL) Specifies an individual Rush subspace to check, requiring versions to be ' +
+        'consistent only within that subspace (ignoring other subspaces). If this parameter is omitted, ' +
+        'then "rush check" will instead require consistency across all projects in the monorepo. ' +
+        'Requires the "subspacesEnabled" setting to be true in subspaces.json.'
```

Comment:
@william2958 the design notes says:


> - `ensureConsistentVersions` always applies within a subspace; there is no mode where it applies across all projects in the monorepo

>- `--subspace` is mandatory for `rush check` when the feature is enabled

That seems like not what is implemented here.

Reply:
Updated help message